app/assets/stylesheets/components/_account.scss

```diff
@@ -270,6 +270,8 @@ div.header-selector {
             margin-right: 10px;
         }
     }
+
+    margin-bottom: 14px;
```

Comment:
This class is used in other parts of the app, have this been accounted for and checked that it doesn't do weird things?

Used in the following files
- AccountOrders
- AccountOverview
- AccountVoting
- AssetWhitelist
- RecentTransactions
- MarketsTable